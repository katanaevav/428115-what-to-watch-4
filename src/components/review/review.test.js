import React from "react";
import renderer from "react-test-renderer";
import Review from "./review.jsx";

const MovieReview = {
  id: 0,
  text: `Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.`,
  author: `Kate Muir`,
  date: Date.parse(`June 3, 2019`),
  mark: 5,
};

it(`Render MovieReview`, () => {
  const tree = renderer.create(
      <Review
        text = {MovieReview.text}
        author = {MovieReview.author}
        date = {MovieReview.date}
        mark = {MovieReview.mark}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
