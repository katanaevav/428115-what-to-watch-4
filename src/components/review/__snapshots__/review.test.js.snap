// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render App 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        Kate Muir
      </cite>
      <time
        className="review__date"
        dateTime="2019-5-3T0:0"
      >
        June 3, 2019
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    5
  </div>
</div>
`;
